### PR TITLE
Add general scenario example to grant

### PR DESCRIPTION
**Pull request by evelynzhaojie**

### CODE REVIEW

**Review by wangsimo0 — DISMISSED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

File: sql-reference/sql-statements/account-management/GRANT.md [hunk 1]
```diff
@@ -49,78 +51,74 @@ GRANT
...
        ｜ ALL FUNCTIONS } IN 
-           { DATABASE <database_name> | ALL DATABASES }
+           { DATABASE <db_name> | ALL DATABASES }
     TO { ROLE | USER} {<role_name>|<user_identity>} [ WITH GRANT OPTION ]
     
-*注意：需要执行 set catalog 之后才能使用。
-function 还可以用 db.function 的方式来进行表示。
-GRANT <priv> ON FUNCTION db.function TO {ROLE <rolename> | USER <username>}
+*注意：需要执行 SET CATALOG 之后才能使用。function 还可以用 db.function 的方式来进行表示。
+GRANT <priv> ON FUNCTION db.function TO {ROLE <role_name> | USER <user_name>}
```
Comment (EsoragotoSpirit) [inline sql-reference/sql-statements/account-management/GRANT.md:121]: `<db_name>.<function_name>`?

File: sql-reference/sql-statements/account-management/GRANT.md [hunk 2]
```diff
@@ -49,78 +51,74 @@ GRANT
...
        ｜ ALL MATERIALIZED VIEWS } IN 
-           { DATABASE <database_name> | ALL DATABASES }
+           { DATABASE <db_name> | ALL DATABASES }
     TO { ROLE | USER} {<role_name>|<user_identity>} [ WITH GRANT OPTION ]
     
-* 注意：需要执行 set catalog 之后才能使用。 
-mv 还可以用 db.mv 的方式来进行表示。
-GRANT <priv> ON MATERIALIZED_VIEW db.mv TO {ROLE <rolename> | USER <username>}
+* 注意：需要执行 SET CATALOG 之后才能使用。 物化视图还可以用 db.mv 的方式来进行表示。
+GRANT <priv> ON MATERIALIZED_VIEW db.mv TO {ROLE <role_name> | USER <user_name>}
```
Comment (EsoragotoSpirit) [inline sql-reference/sql-statements/account-management/GRANT.md:109]: `<db_name>.<mv_name>`

File: sql-reference/sql-statements/account-management/GRANT.md [hunk 3]
```diff
@@ -49,78 +51,74 @@ GRANT
...
        ｜ ALL VIEWS } IN 
-           { DATABASE <database_name> | ALL DATABASES }
+           { DATABASE <db_name> | ALL DATABASES }
     TO { ROLE | USER} {<role_name>|<user_identity>} [ WITH GRANT OPTION ]
     
-*注意：需要执行 set catalog 之后才能使用。
-注意：view 还可以用 db.view 的方式来进行表示。
-GRANT <priv> ON VIEW db.view TO {ROLE <rolename> | USER <username>}
+*注意：需要执行 SET CATALOG 之后才能使用。view 还可以用 db.view 的方式来进行表示。
+GRANT <priv> ON VIEW db.view TO {ROLE <role_name> | USER <user_name>}
```
Comment (EsoragotoSpirit) [inline sql-reference/sql-statements/account-management/GRANT.md:97]: `<db_name>.<view_name>`

File: sql-reference/sql-statements/account-management/GRANT.md [hunk 4]
```diff
@@ -49,78 +51,74 @@ GRANT
...
        | ALL TABLES } IN 
-           { DATABASE <database_name> | ALL DATABASES }
+           { DATABASE <db_name> | ALL DATABASES }
     TO { ROLE | USER} {<role_name>|<user_identity>} [ WITH GRANT OPTION ]
 
-*注意：需要执行 set catalog 之后才能使用。
-注意：table还可以用db.tbl的方式来进行表示
-GRANT <priv> ON TABLE db.tbl TO {ROLE <rolename> | USER <username>}
+*注意：需要执行 SET CATALOG 之后才能使用。table 还可以用 db.tbl 的方式来进行表示。
+GRANT <priv> ON TABLE db.tbl TO {ROLE <role_name> | USER <user_name>}
```
Comment (EsoragotoSpirit) [inline sql-reference/sql-statements/account-management/GRANT.md:85]: `<db_name>.<table_name>`

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by evelynzhaojie** [comment]
@mergify backport 3.1 3.0

**Pull request merged**